Ipc4 sdw

bardliao opened the pull request.

bardliao committed:
topology2: add cavs sdw topology

Add sdw topology for headset which includes one playback and one capture
pcms.

Signed-off-by: Bard Liao <yung-chuan.liao@linux.intel.com>

RanderWang commented on tools/topology/topology2/cavs-sdw.conf:
```diff
@@ -0,0 +1,127 @@
+
+		# include DAI copier components
+		Object.Widget.copier."0" {
+			index 2
```
+ dai_index 2
dai_type "ALH"

RanderWang replied:
also the following one

bardliao committed:
topology2: add ALH DAI class

Add ALH DAI class definition, ALH DAI objects can be
instantiated as:

Object.Dai.ALH."N" {
 	direction		"capture" # playback or capture
 	dai_name		"SDW0-Capture"
 	id 			2
 	Object.hw_config."2" {
 	}
}

Signed-off-by: Bard Liao <yung-chuan.liao@linux.intel.com>

RanderWang committed:
Merge branch 'ipc4_dmic' into ipc4_sdw
